### PR TITLE
Bare flask + container skeleton

### PR DESCRIPTION
**Pull request by jpichon**
Update the structure to match that of the other services.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,15 +2,20 @@
 # This file will be regenerated if you run travis_pypi_setup.py
```
Comment (fuzzball81) [inline .travis.yml:2]: I wonder if we should remove this and instead use github actions similar to https://github.com/release-depot/flask-container-scaffold/blob/main/.github/workflows/test.yml

**Review by fuzzball81 — COMMENTED** [review]
Overall this looks good. We will likely use scaffold at some point, but this gets us up and running.

File: .travis.yml (hunk 1 above)
Comment (jguiditta) [reply .travis.yml:2]: Agree, travis has not been great recently, and I think actions are now the preferred method

**Comment by jguiditta** [comment]
ACK, aside from the minor point @fuzzball81 made about using actions instead of travis now.  It should be pretty straight forward to port over what I have in the referenced project, but if there is a complication, I can help.

File: .travis.yml (hunk 1 above)
Comment (jpichon) [reply .travis.yml:2]: I agree with the idea as well. I tried to update this PR with a first attempt at a github action, though I wouldn't be surprised if it takes a couple iterations to get it working!

Comment (jguiditta) [reply .travis.yml:2]: I think what you have looks fine, we can always do a rev to fix CI if needed.  Are you wanting to leave travis file in until we see if the new actions work correctly? If so, we could easily test the action with the next patch removing travis (the other alternative would be to temporarily not limit push action tests to be on main (well, I guess this is still using 'master'), and once you see the action work on your own branch, you will know you can limit it to just main for the patch).

Comment (jpichon) [reply .travis.yml:2]: Sounds reasonable to me! And yes, the repo was created a couple of years ago so it still uses master.

Comment (jguiditta) [reply .travis.yml:2]: ok - maybe as a next step before adding more working code, we should move this to the newer 'main' standard?  We need to start shifting everything anyway.  I am approving this, looks good!

**Review by jguiditta — APPROVED** [review]

**Comment by jpichon** [comment]
Looks like we need to set this up with mergify as well, in addition to checking the GH action works + updating the branch name. Opened https://github.com/release-depot/rtls/issues/5 to keep track of it.

**Comment by jguiditta** [comment]
> Looks like we need to set this up with mergify as well, in addition to checking the GH action works + updating the branch name. Opened #5 to keep track of it.

Right you are, manually merging now!

**Pull request merged**